VocaDbWeb/VocaDbWeb.csproj

```diff
@@ -42,7 +42,7 @@
     <DebugType>full</DebugType>
     <Optimize>false</Optimize>
     <OutputPath>bin\</OutputPath>
-    <DefineConstants>DEBUG;TRACE</DefineConstants>
+    <DefineConstants>TRACE;DEBUG;NETFRAMEWORK</DefineConstants>

```

Comment:
The `NETFRAMEWORK` constant must be removed after we finished migrating to .NET Core.